code/go/ecs/file.go

```diff
@@ -34,9 +34,19 @@ type File struct {
 	Name string `ecs:"name"`
 
 	// Directory where the file is located.
+	// On platforms with drive letters, the drive letter should not be part of
+	// the directory field. For example, "C:\\Program Files\\Foo" should be
+	// parsed as `file.directory` "Program Files\\Foo" and `drive.letter` "C".
 	Directory string `ecs:"directory"`
 
-	// Full path to the file.
+	// Drive letter where the file is located.
+	Drive string `ecs:"drive"`
```

Comment:
Should this bee `drive_letter`? I think there's a difference, like for . mounted vs unmounted in Windows

Reply:
Oh jeez yeah, good point 🤦‍♂ 😂